src/config.ts

```diff
@@ -109,11 +110,11 @@ const mumbai: Config = {
       name: 'Matic',
       decimals: 18,
     },
-    '0xe6b8a5CF854791412c1f6EFC7CAf629f5Df1c747': {
```

Comment:
Why deleting this one ?

Reply:
Wasn't working for me, I don't think it's been approved. I can put it back.